src/lib/biokepi_pipeline.ml

```diff
@@ -99,6 +100,13 @@ module Construct = struct
     let params = {gap_open_penalty; gap_extension_penalty} in
     Bwa (params, fastq)
 
+  let bwa_mem
+      ?(gap_open_penalty=Bwa.default_gap_open_penalty)
+      ?(gap_extension_penalty=Bwa.default_gap_extension_penalty)
+      fastq =
+    let params = {gap_open_penalty; gap_extension_penalty} in
+    Bwa_mem (params, fastq)
```

Comment:
I initially put used to 2 parameters `{gap_open_penalty; gap_extension_penalty}` as a “demo”

Do you think they are really relevant?
Which ones should we really export there?